cloudinit/distros/openbsd.py

```diff
@@ -25,7 +25,7 @@ def _write_hostname(self, hostname, filename):
     def _get_add_member_to_group_cmd(self, member_name, group_name):
         return ["usermod", "-G", group_name, member_name]
 
-    def manage_service(self, action: str, service: str):
+    def manage_service(self, action: str, service: str, rcs=None):
```

Comment:
Forgot the `@classmethod`

Reply:
I see it in the latest version of this branch. I'm guessing this got fixed in a later commit than the one that you reviewed.